lms/djangoapps/onboarding/forms.py

```diff
@@ -362,6 +362,7 @@ def save(self, request, commit=True):
         user = user_info_survey.user
         first_name = user.first_name
         last_name = user.last_name
+        user_info_survey.is_first_learner = False
```

Comment:
What will happen in this scenario?

If user UA created an organization OA, Now this method is called to update user_info_survey attributes other then organization, This implementation will change is_first_learner to False, But actually it should not be set to False in this case. as user was first_learner and didn't change the organization. He should be first learner.